src/App.js

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
+import SearchBar from './containers/SearchBar';
+import WeatherList from './containers/WeatherList';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
do you really need to import the bootstrap css here ? the path of `node_modules` on production environment will be different and it will crash. I believe the bootstrap css will automatically get involved when you added the react-bootstrap as a dependency and using the class names and components of that package.

Reply:
I have removed the import statement and included a stylesheet link to the index.html file as instructed by the [react-bootstrap guide](https://react-bootstrap.github.io/getting-started.html)